charts/kong/templates/deployment.yaml

```diff
@@ -93,9 +93,14 @@ spec:
         resources:
 {{ toYaml .Values.resources | indent 10 }}
         command:
-        - "rm"
-        - "-vrf"
-        - "$KONG_PREFIX/pids"
+        - sh
+        - -c
+        - 'if test -S $KONG_PREFIX/worker_events.sock; then
```

Comment:
Is there a reason to use the `test -S` rather than having this also use an `rm -vf` to not fail if the file isn't present? It would technically remove non-socket files also, but there's no reason that this path should ever be anything else.

Removing the if case and running the same set of commands always would make this simpler.

Reply:
I had tried that initially but it was failing (maybe because of some other issue in my system). Since this issue has already addressed I'm planning to upgrade the images. I'm closing this request, Thank You for your support.